sporkfish/evaluator/evaluator.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC, abstractmethod
+
+
+class Evaluator(ABC):
```

Comment:
Sorry to be a pain, could we add some base class comments like you have for Pesto?